client/src/components/EmailFrontend.js

```diff
@@ -167,8 +167,27 @@ class EmailFrontend extends React.Component {
       )
       .value();
 
+    const all_connected_user_fields_are_filled = _.chain(user_fields)
+      .toPairs()
+      .filter(([field_name, field_val]) =>
+        _.includes(_.keys(field_val), "connection")
```

Comment:
I'm guessing here but is it necessary to put two filters here?
```
      .filter(
        ([field_name, field_val]) =>
          _.includes(_.keys(field_val), "connection") &&
          _.includes(_.keys(completed_template), field_val.connection.name) &&
          _.includes(
            completed_template[field_val.connection.name],
            field_val.connection.enable
          )
      )
```
You should check if that works !